packages/server/api/src/app/workers/redis/redis-handler.ts

```diff
@@ -0,0 +1,33 @@
+import { getRedisConnection } from '../../database/redis-connection'
+
+let redis = getRedisConnection() 
+
+export const redisHandler = {
+    async init(): Promise<void> {
+        redis = getRedisConnection()
+    },
+
+    async push(userId: string, jobId: string): Promise<void> {
+        const userKey = `active_runs:${userId}`
+        await redis.sadd(userId, jobId)
+        await redis.incr(userKey)
```

Comment:
For example what if we had a bug that this counter is wrong and we had to wrong, It will not self correct it's self, I would add TTL for the record but not sure if this is optimal answer
```ts
        await redis.expire(userKey, TTL_SECONDS) // Reset TTL on access
```